var/spack/repos/jcsda-emc-bundles/packages/jedi-fv3-env/package.py

```diff
@@ -12,11 +12,10 @@ class JediFv3Env(BundlePackage):
     homepage = "https://github.com/JCSDA/fv3-bundle"
     git = "https://github.com/JCSDA/fv3-bundle.git"
 
-    maintainers = ["climbfuji", "rhoneyager"]
+    maintainers = ["climbfuji", "srherbener"]
 
     version("1.0.0")
 
-    depends_on("base-env", type="run")
```

Comment:
Was this already there and wasn't necessary, or is this related to jedi-base-env changes?

Reply:
That's already included in jedi-base-env as a dependency, so not needed here.